src/modules/components/vote/voteProposalDataListItem/voteProposalDataListItemStructure/voteProposalDataListItemStructure.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import { DataList, Tag, type IDataListItemProps } from '../../../../../core';
+import { voteIndicatorToTagVariant } from '../../voteDataListItem/voteDataListItemStructure/voteDataListItemStructure';
+
+export interface IVoteProposalDataListItemStructureProps extends IDataListItemProps {
+    /**
+     * The ID of proposal.
+     */
+    id: string;
+    /**
+     * The account details of the voter.
+     */
+    title: string;
+    /**
+     * The vote of the user.
+     */
+    voteIndicator: 'yes' | 'no' | 'abstain' | 'approve';
+    // TO-DO: implement formatter decision
+    /**
+     * The Unix timestamp of the transaction.
+     */
+    date?: string;
```

Comment:
Let's just be explicit and say that we're expecting a formatted date and, when we'll have the formatter, we'll change these comments and use the formatter internally
```suggestion
    /**
     * Formatted date of the user's vote on the proposal
     */
    date?: string;
```